### ISSUE TITLE
Can the model work well on fetal dataset?

### ISSUE THREAD

**Issue by Noob-PeNguiN** [BBillot/SynthSeg#118]
I tried running the pre-trained SynthSeg 2.0 model on the FeTA fetal brain dataset. As shown in the image below, the model struggles to produce a meaningful segmentation.

This is understandable, as the current model is optimized for adult neuroanatomy. I am now considering training a dedicated model for fetal MRI using the SynthSeg framework, and I would love to get your perspective on a few points:

- In your experience, does the SynthSeg generative strategy adapt well to the specific morphology and contrast of fetal brains?
- Are there any specific hyperparameters or augmentation settings you would suggest adjusting for training on fetal MRI?

Thank you for your time and for your cool work!

Best regards.

<img width="737" height="636" alt="Image" src="https://github.com/user-attachments/assets/5173fafd-cc46-4fae-86d9-b585cc2438bf" />

**Comment by xychen2022** [comment]
@Noob-PeNguiN Were you able to train your model successfully? I’m currently training a similar model and was wondering how long it typically takes to achieve decent performance. @BBillot I also wanted to ask whether there are any specific hyperparameters or data augmentation settings you would recommend tuning during training. In particular, is using only GMM-based augmentation sufficient, or would you suggest incorporating additional augmentations? Thanks!

**Comment by BenBillot** [comment]
Hi,
it's unlikely that the baseline SynthSeg model can segment fetal data out of the box, because the anatomy deviates too much from the training adult population. 
But sorry, I haven't yet worked on training a fetal SynthSeg, so it's exploratory work from there.
Good luck!
Benjamin

**Comment by BenBillot** [comment]
PS: could you please close the issue? I lost access to my previous github account, so I cannot close issues anymore. Thanks! :)

**Comment by xychen2022** [comment]
> PS: could you please close the issue? I lost access to my previous github account, so I cannot close issues anymore. Thanks! :)

Before closing, can you let know how many training steps are needed to train a good enough model?

**Comment by Noob-PeNguiN** [comment]
@BenBillot Thanks for the reply! I think so too, so I may try it some other time since segmentation is not what I mainly study right now.
@xychen2022 I haven't tried it yet. I trained my data with nn-Unet on feta dataset (which has around 80 stacks) with default settings and it works fine. It takes about 18 hours to train it on A6000. About the training steps for synthseg, I recommend you to try some different training steps and print out the result and loss so that you can know. If you have futher question you can open another issue : )
